### PR TITLE
Convert kubernetes-e2e-upgrades-image-gke 1.3-->1.5 jobs

### PR DESCRIPTION
**Pull request by xiangpengzhao**
#1230 #1866 #1871 

/cc @krzyzacy

### CODE REVIEW

File: jobs/ci-kubernetes-e2e-gke-container_vm-1.3-container_vm-1.5-upgrade-cluster-new.env [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+JENKINS_PUBLISHED_SKEW_VERSION=ci/latest-1.5
+JENKINS_PUBLISHED_VERSION=ci/latest-1.3
+JENKINS_USE_SKEW_TESTS=true
+KUBE_GKE_IMAGE_TYPE=container_vm
+PROJECT=gke-up-c1-3-c1-5-up-clu-n
+ZONE=us-central1-c
+
+### version-env
+# 1.3 doesn't support IAM, so we should use cert auth.
+CLOUDSDK_CONTAINER_USE_CLIENT_CERTIFICATE=true
```
Comment (xiangpengzhao) [inline jobs/ci-kubernetes-e2e-gke-container_vm-1.3-container_vm-1.5-upgrade-cluster-new.env]: Should the version-env be kept? @krzyzacy

Comment (krzyzacy) [inline jobs/ci-kubernetes-e2e-gke-container_vm-1.3-container_vm-1.5-upgrade-cluster-new.env]: yes!

**Comment by krzyzacy** [comment]
@k8s-bot ok to test

File: jobs/config.json [hunk 2]
```diff
@@ -2579,6 +2579,104 @@
...
+
+"ci-kubernetes-e2e-gke-container_vm-1.3-container_vm-1.5-upgrade-master": {
+  "scenario": "kubernetes_e2e",
+  "args": [
+    "--env-file=platforms/gke.env",
+    "--env-file=jobs/ci-kubernetes-e2e-gke-container_vm-1.3-container_vm-1.5-upgrade-master.env"
+  ]
+},
+
+"ci-kubernetes-e2e-gke-container_vm-1.3-container_vm-1.5-upgrade-clusterr": {
```
Comment (krzyzacy) [inline jobs/config.json]: clusterr :-)

Comment (xiangpengzhao) [inline jobs/config.json]: Ohhhh! I made the same mistake twice!!! :sob:

**Comment by xiangpengzhao** [comment]
Addressed inline comments. Also delete the redundant `ZONE=us-central1-c`. PTAL @krzyzacy

**Pull request merged**